.github/workflows/cicd.yaml

```diff
@@ -47,67 +48,15 @@ jobs:
           tox
 
       - name: tests
-        env:
-          ENABLE_BDIST_EXT_MODULE: 'yes'
-          TOXENV: python${{ matrix.python-version }}
```

Comment:
Does tox run over the specified versions [here](https://github.com/argoproj-labs/hera-workflows/blob/main/tox.ini#L2)? Also, will it automatically run `testenv` in this case?

Reply:
But, this maybe confusing for the users who are not familiar with tox and github actions

Reply:
I agree with you! I'll fix this.